kmip/core/messages/contents.py

```diff
@@ -118,6 +118,40 @@ def __ne__(self, other):
         else:
             return NotImplemented
 
+    def __lt__(self, other):
+        if isinstance(other, ProtocolVersion):
+            if self.protocol_version_major < other.protocol_version_major:
+                return True
+            elif self.protocol_version_minor < other.protocol_version_minor:
```

Comment:
Do you need to add self.protocol_version_major == other.protocol_version_major to this elif statement? At this point we know that self.protocol_version_major >= other.protocol_version_major. What if self protocol version is 2.1 and other's is 1.2? I think this would report True for 2.1 < 1.2.